src/cdm.js

```diff
@@ -0,0 +1,82 @@
+import window from 'global/window';
+import videojs from 'video.js';
+
+const keySystems = {
+  fairplay: 'com.apple.fairplay',
+  playready: 'com.microsoft.playready',
+  widevine: 'com.widevine.alpha',
+  clearkey: 'org.w3.clearkey'
+};
+
+// Use a combination of API feature and user agent detection to provide an initial
+// best guess as to which CDMs are supported.
+const bestGuessSupport = {
+  fairplay: !!window.WebKitMediaKeys,
+  playready: !!(window.MSMediaKeys && videojs.browser.IE_VERSION) ||
+    !!(window.MediaKeys && window.navigator.requestMediaKeySystemAccess && videojs.browser.IS_EDGE),
```

Comment:
should we note here that this won't catch the new chromium edge?

Reply:
Never mind, we will have it if you'll use a new-enough Video.js verison (https://github.com/videojs/video.js/pull/6497)